### DISCUSSION TITLE
在对象生成的表单里怎么隐藏名称那个字段

### DISCUSSION THREAD

**Discussion by hengyanqwe** [steedos/steedos-platform#5614] [Q&A]
![image](https://github.com/steedos/steedos-platform/assets/68996932/3eb9aaeb-612c-47bb-99c2-2982fa2953d1)

**Comment by hengyanqwe** [comment]
并且这个对象字段怎么也删不了
![image](https://github.com/steedos/steedos-platform/assets/68996932/921137a4-9b92-41b7-8e56-efa310dd2b97)

**Comment by rttmoa** [comment]
数据库里有缓存吧 把缓存删掉
